src/Weather_forecast_on_dashboard.py

```diff
@@ -0,0 +1,133 @@
+import streamlit as st
```

Comment:
you need to delete this file too, and you have added weather_forecast.py. Next time you rename a file, lets do it together on git, and Ill show you a nice way